bool thetaCommand
int8 theta
bool distanceCommand
float32 distance
